samples/can/Kconfig

```diff
@@ -0,0 +1,93 @@
+# Kconfig - Private config options for can sample app
+
+#
+# Copyright (c) 2018 Alexander Wachter
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+mainmenu "Controller Area Network sample application"
+
+config ZEPHYR_BASE
+	string
+	option env="ZEPHYR_BASE"
+
+source "$ZEPHYR_BASE/Kconfig.zephyr"
+
+config CAN_DEV
+	string "Name of the CAN device"
+	default "CAN_1"
+	help
+	  Name of the can device used for send an receive.
+
+config GPIO_LED_DEV
+	string "Name of the LED GPIO port"
+	default "GPIOC"
+	help
+	  Name of the LED port for signaling message reception.
+
+config GPIO_BUTTON_DEV
+	string "Name of the button GPIO port"
+	default "GPIOA"
+	help
+	  Name of the button port for triggering messages.
+
+config PIN_USER_BUTTON
+	int "Pin User Button"
+	default 0
+	help
+	  Pin number of the user Button.
+
+config PIN_LED_1
+	int "Pin LED 1"
+	default 6
+	help
+	  Pin number of the first LED.
+
+config USE_LED_2
```

Comment:
I don't see how option named "USE_LED_2" can have a description of "Use two LEDs". "USE_LED_2" clearly means "use LED no.2". Please update option names for better clarity.

Actually, I don't see why boolean options are used. If user can select 1-4 LEDs, then it should be an integer option with the given range.

But the best of all would be following: if this is the basic CAN example, which can work with just 1 LED, please just make it so - the source code will be much more clear (important, because many people will use it as a reference for CAN things), and it will work on as many boards as possible without extra configuration efforts.

Thanks.

Reply:
Done.

Reply:
Thanks!